cache.go

```diff
@@ -68,42 +68,49 @@ func New() Operations {
 		MemgodbInstance: Memgodb,
 	}
 
-	c := cron.New()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
 
-	// cron job set to run every 1 minute
-	c.AddFunc("*/1 * * * *", func() {
-		if debug {
-			logger.Info().Msg("cron job running...")
-		}
+	go func() { // launch "cron job"
+		for {
+			// wait for context cancellation
+			select {
+			case <-ctx.Done():
```

Comment:
The goroutine ends here, preventing the subsequent code blocks from running.

Reply:
#51 for reference